frontend/dockerfile/linter/generate.go

```diff
@@ -30,7 +30,7 @@ type Rule struct {
 
 const tmplStr = `---
 title: {{ .Rule.Name }}
-description: {{ .Rule.Description }}
+description: "{{ .Rule.Description }}"
```

Comment:
I think we could use a block scalar style folded with no newline instead:

```suggestion
description: >-
  {{ .Rule.Description }}
```

in case double-quotes are also part of the description?

Reply:
Sure, I thought about it too, but I thought it was unlikely to happen. But you're right this is safer.